script/spider.py

```diff
@@ -61,22 +60,54 @@ def getTag(soup):
     return tagStr
 
 
+def getParentText(soup):
+    if soup != None:
+        return soup.parent.text.strip()
+
+    return ''
+
 def getDetail(soup):
 
     name= soup.find('a',class_='d-inline-block').text.strip()
-    language_dom= soup.find('span',class_='mr-3')
-    if language_dom != None:
-        language = language_dom.text.strip()
-        return name, language
-    else:
-        return name,''
-    
+    description = soup.find('p',class_ = 'wb-break-word').text.strip()
+    programmingLanguage = getParentText(soup.find('span',class_="repo-language-color"))
+    issue =getParentText(soup.find('svg',class_="octicon-issue-opened"))
+    star = getParentText(soup.find('svg',class_="octicon-star"))
+    pr = getParentText(soup.find('svg',class_= "octicon-git-pull-request"))
+    fork = getParentText(soup.find('svg',class_="octicon-repo-forked"))
+
+    return name, {
+        "description":description,
+        "programmingLanguage":programmingLanguage,
+        "issue":issue,
+        "star":star,
+        "pr":pr,
+        "fork":fork
+    }
 
 
-def Merge(dict1, dict2): 
+def Merge(dict1, name_str, dict2): 
```

Comment:
Please add some comments on the Merge method. It's hard to understand the code without comments.